java/code/src/com/suse/manager/webui/controllers/contentmanagement/mappers/ResponseMappers.java

```diff
@@ -63,6 +63,26 @@ public static ProjectPropertiesResponse mapProjectPropertiesFromDB(ContentProjec
         return properties;
     }
 
+    /**
+     * Map a list of db sources entities to a list of view response beans
+     *
+     * @param sourcesDB the list of db envs
+     * @return the List<EnvironmentResponse> view beans
+     */
+    public static List<ProjectSourceResponse> mapSourcesFromDB(List<ProjectSource> sourcesDB) {
+        return sourcesDB
+                .stream()
+                .map(sourceDb -> {
+                    ProjectSourceResponse projectSourceResponse = new ProjectSourceResponse();
+                    projectSourceResponse.setSourceLabel(sourceDb.getId().toString());
```

Comment:
This was probably intended to be `setSourceId`

Reply:
:+1: